per/models.py

```diff
@@ -656,3 +657,32 @@ class PerWorkPlan(models.Model):
 
     def __str__(self):
         return f"{self.overview.id}"
+
+
+class LearningType(models.IntegerChoices):
+    LESSON_LEARNED = 1, _('Lesson learned')
+    CHALLENGE = 2, _('Challenge')
+
+
+@reversion.register()
+class OpsLearning(models.Model):
+    learning = models.TextField(verbose_name=_("learning"), max_length=900, null=True, blank=True)
```

Comment:
Is there a reason for the `max_length` here? a `max_length` for CharFields (which get created as `varchar` in the db) is required, but for `TextField` which gets created as a `text` field in postgres, I don't think there's any performance reasons to add a `max_length` - the only thing (that I'm aware of) this would really do is add validation in the Admin to not allow people to save anything over 900 chars.

If there's a business requirement for that, that's great, otherwise let's not add unneeded constraints.

Reply:
Done, 900 chars limit removed.